Cargo.toml

```diff
@@ -15,8 +15,11 @@ edition = "2018"
 
 [features]
 
-default = ["sync"]
+default = ["sync", "native-tls"]
 sync = ["tokio"]
+native-tls = ["reqwest/default-tls"]
```

Comment:
Should we maybe call it `default-tls` as well in case reqwest ever changes the default?

Reply:
On a 2nd thought, it will be a breaking change, so we'll have to adapt anyway.